predicators/planning.py

```diff
@@ -1156,9 +1213,17 @@ def run_task_plan_once(
         assert "FD_EXEC_PATH" in os.environ, \
             "Please follow instructions in the docstring of the" +\
             "_sesame_plan_with_fast_downward method in planning.py"
-        if CFG.sesame_task_planner == "fdopt":
+
+        sesame_task_planner = CFG.sesame_task_planner
+        if sesame_task_planner.endswith("-costs"):
+            use_costs = True
+            sesame_task_planner = sesame_task_planner[:-len("-costs")]
+        else:
+            use_costs = False
```

Comment:
seems to me like currently cost-aware planning only works with FD. Shouldn't we assert that the `CFG.sesame_task_planner` is set to FD then to prevent any issues?

Reply:
we're already within `elif "fd" in CFG.sesame_task_planner:` here. if someone tries to do something like `--sesame-task-planner astar-costs`, it will just the usual NotImplementedError at the end of the outer conditional statement